packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_destroy_loan_scheme.test.ts

```diff
@@ -0,0 +1,239 @@
+  const testing = Testing.create(container)
+
+  let providers: MockProviders
+  let builder: P2WPKHTransactionBuilder
+
+  beforeAll(async () => {
+    await testing.container.start()
+    await testing.container.waitForWalletCoinbaseMaturity()
+
+    providers = await getProviders(testing.container)
+    providers.setEllipticPair(WIF.asEllipticPair(GenesisKeys[GenesisKeys.length - 1].owner.privKey))
+    builder = new P2WPKHTransactionBuilder(providers.fee, providers.prevout, providers.elliptic, RegTest)
+
+    // Default scheme
+    await testing.container.call('createloanscheme', [100, new BigNumber(1.5), 'default'])
+    await testing.generate(1)
+  })
+
+  afterAll(async () => {
+    await testing.container.stop()
+  })
+
+  beforeEach(async () => {
+    // Fund 10 DFI UTXO
+    await fundEllipticPair(testing.container, providers.ellipticPair, 10)
+    await providers.setupMocks() // Required to move utxos
+  })
+
+  afterEach(async () => {
+    const data = await testing.container.call('listloanschemes')
+    const result = data.filter((d: { default: boolean }) => !d.default)
+
+    for (let i = 0; i < result.length; i += 1) {
+      // Delete all schemes except default scheme
+      await testing.container.call('destroyloanscheme', [result[i].id])
+      await testing.generate(1)
+    }
+  })
+
+  it('should destroyLoanScheme', async () => {
+    await testing.container.call('createloanscheme', [200, new BigNumber(2.5), 'scheme'])
+    await testing.generate(1)
+
+    // Before delete
+    {
+      const data = await testing.container.call('listloanschemes')
+      const result = data.filter((d: { id: string }) => d.id === 'scheme')
+      expect(result.length).toStrictEqual(1)
+    }
+
+    const script = await providers.elliptic.script()
+    const txn = await builder.loans.destroyLoanScheme({
+      identifier: 'scheme'
+    }, script)
+
+    // Ensure the created txn is correct
+    const outs = await sendTransaction(testing.container, txn)
+    expect(outs[0].value).toStrictEqual(0)
+    expect(outs[1].value).toBeLessThan(10)
+    expect(outs[1].value).toBeGreaterThan(9.999)
+    expect(outs[1].scriptPubKey.addresses[0]).toStrictEqual(await providers.getAddress())
+
+    // Ensure you don't send all your balance away during destroy loan scheme
+    const prevouts = await providers.prevout.all()
+    expect(prevouts.length).toStrictEqual(1)
+    expect(prevouts[0].value.toNumber()).toBeLessThan(10)
+    expect(prevouts[0].value.toNumber()).toBeGreaterThan(9.999)
+
+    // After delete
+    {
+      const data = await testing.container.call('listloanschemes')
+      const result = data.filter((d: { id: string }) => d.id === 'scheme')
+      expect(result.length).toStrictEqual(0)
+    }
+  })
+
+  it('should not destroyLoanScheme if identifier is more than 8 chars long', async () => {
+    const script = await providers.elliptic.script()
+    const txn = await builder.loans.destroyLoanScheme({
+      identifier: '123456789'
```

Comment:
```suggestion
      identifier: '1'.repeat(9)
```
Might be clearer to express a string length this way from now on

Reply:
Just fixed and pushed